criu/cgroup.c

```diff
@@ -716,32 +716,31 @@ static int collect_cgroups(struct list_head *ctls)
 	return 0;
 }
 
-int dump_task_cgroup(struct pstree_item *item, u32 *cg_id, struct parasite_dump_cgroup_args *args)
+int dump_task_cgroup(struct pid *id, u32 *cg_id, struct parasite_dump_cgroup_args *args)
```

Comment:
This function previously passed in `pstree_item` to check if it was NULL. If the item is NULL, then its pid should also be NULL. Therefore, it is sufficient to pass in its pid instead.